contact-track/src/App.js

```diff
@@ -0,0 +1,129 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import {
+  Table,
+  TableBody,
+  TableHeader,
+  TableHeaderColumn,
+  TableRow,
+  TableRowColumn,
+} from 'material-ui/Table';
+
+import ReactTable from 'react-table'
+import 'react-table/react-table.css'
```

Comment:
should organize your imports to keep things maintainable, let's put this near the `App.css` import